Previews: update docs for the new search layout (#50130)
grafana/grafana: docs/sources/dashboards/previews.md

> **Note:** Dashboard previews are available in Grafana 9.0+ as an opt-in beta feature. Data source permissions are not yet taken into the account when displaying the dashboard previews - refer to the [permissions]({{< relref "#preview-visibility">}}) to learn more before enabling the feature.

> You can view dashboard previews only when the feature is enabled for your instance. It is an opt-in feature that is, by default, disabled. You can view the previews in the Grafana UI after selecting the new grid layout once the administrator enables the feature following the procedure described below.

Reviewer: ```suggestion
You can view dashboard previews after the administrator enables the feature for your instance and after you select the new grid layout. The dashboard previews feature is an opt-in feature that is disabled by default. 
```

Author: The original text also has this: `once the administrator enables the feature following the procedure described below.` to differentiate between doing the whole setup (enable feature flag+setup renderer process) and just enabling the feature flag.

Wdyt about replacing `for your instance` with `following the procedure described below` in your suggestion? I think _for your instance_ (rather than anyone elses instance) is implied
